frontend/heart-code/src/components/profile-page/PersonalPage.tsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import { User } from "../../models/User";
+import { useApi } from "../../hooks";
+import Profile from "./Profile";
+// import { useParams } from "react-router-dom";
+
+export default function PersonalPage() {
+  const [user, setUser] = useState<User>();
+  const { get } = useApi();
+
+  useEffect(() => {
+    get<User>("users/account").then((response) => {
+      setUser(response.data);
+    });
+  }, []);
+
+  //_______________________________________________________
+  //   const { userID } = useParams();
+  //   const [user, setUser] = useState<User>();
+  //   useEffect(() => {
+  //     get<User>(`users/${userID}`).then((response) => {
+  //       setUser(response.data);
+  //     });
+  //   }, []);
+  //_______________________________________________________
```

Comment:
comments?

Reply:
Oh, ja dat is voor als we straks profielen van andere mensen kunnen bekijken, alleen dat zou dan op de home page moeten, niet de personal page, kan mss wel weg voor nu